modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -44,16 +45,16 @@ func NewKeeperWithVM(
 		panic(errors.New("wasm VM must be not nil"))
 	}
 
-	if types.WasmVM != nil && !reflect.DeepEqual(types.WasmVM, vm) {
+	if globals.WasmVM != nil && !reflect.DeepEqual(globals.WasmVM, vm) {
```

Comment:
I wanted to remove the check `!reflect.DeepEqual(globals.WasmVM, vm)` but if I do that, then tests fail because more than one chain is setup and then the condition `globals.WasmVM != nil` will fail. If `WasmVM` is now internal and there is no risk of misusage outside of 08-wasm package, then maybe we can get rid of both checks?